python/tank_vendor/shotgun_authentication/defaults_manager.py

```diff
@@ -68,26 +91,45 @@ def set_host(self, host):
 
     def get_http_proxy(self):
         """
-        Provides the http proxy associated to the default host.
+        Called by the authentication system when it needs to retrieve the 
+        proxy settings to use for a Shotgun connection.
+        
+        The format should be the same as is being used in the Shotgun API.
+        For more information, see the Shotgun API documentation:
+        https://github.com/shotgunsoftware/python-api/wiki/Reference%3A-Methods#shotgun
 
-        :returns: String containing the default http proxy. Default implementation returns None.
+        :returns: String containing the default http proxy.
```

Comment:
Should we let people know what this particular instance of the DefaultsManager doesn't have a proxy? That's why I said "Default implementation returns None."